src/forms/form-fields/geometry-field.js

```diff
@@ -1,9 +1,11 @@
 import { Component } from 'react';
 import PropTypes from 'prop-types'
 import TextField from '../../forms/form-fields/text-field';
-import { isNumber } from 'd2-ui/lib/forms/Validators';
+import Validators from '@dhis2/d2-ui-forms/Validators';
 import { getOr } from 'lodash/fp';
 
+const { isNumber } = Validators
```

Comment:
Validators are exported with

```js
export default {
  isNumber,
  isUrl,
  // ....
}
```

so doing e.g. `import { isNumber } from '@dhis2/d2-ui-forms/Validators'` does not work